src/lib/hooks/use-best-quote/index.ts

```diff
@@ -175,18 +168,15 @@ export const useBestQuote = (
         ) {
           console.log('canRedeemIndexToken')
```

Comment:
just a leftover from old times?

Reply:
yes sir - but good point will add a logs clean up PR some time in the next days.